### ISSUE TITLE
Loading into a 3-node cluster simply stalls.

### ISSUE THREAD

**Issue by srh** [dgraph-io/dgraph#1187]
To reproduce:

Create a fresh directory `~/blah`.  Create `~/blah/groups.conf` with the contents `default: fp % 1 + 1`.  Then run the following commands:

```
mkdir ~/blah/svr{1..3}

~/go/bin/dgraph --group_conf ~/blah/groups.conf --groups "0,1" \
  --idx 1 --port 8080 --grpc_port 9080 --workerport 12345 --my "127.0.0.1:12345" \
  --debugmode=true \
  --p ~/blah/svr1/p --w ~/blah/svr1/w >> ~/blah/svr1/dgraph.log 2>&1 &

~/go/bin/dgraph --group_conf ~/blah/groups.conf --groups "0,1" --idx 2 \
  --port 8081 --grpc_port 9081 --workerport 12346 --my "127.0.0.1:12346" \
  --peer "127.0.0.1:12345" \
  --debugmode=true --p ~/blah/svr2/p --w ~/blah/svr2/w \
  >> ~/blah/svr2/dgraph.log 2>&1 &

~/go/bin/dgraph --group_conf ~/blah/groups.conf --groups "0,1" --idx 3 \
  --port 8082 --grpc_port 9082 --workerport 12347 --my "127.0.0.1:12347" \
  --peer "127.0.0.1:12345" \
  --debugmode=true --p ~/blah/svr3/p --w ~/blah/svr3/w \
  >> ~/blah/svr3/dgraph.log 2>&1 &

```

Then run `~/go/bin/dgraphloader -r ~/1million.rdf.gz -s ~/benchmarks/data/21million.schema`

The result is it hangs on the first request.  It works fine if you just have the first node.

**Comment by manishrjain** [comment]
We've been doing that with Stack Overflow data. It works OK. What's your set up like?

**Comment by srh** [comment]
This worked fine for me recently / a few commits previously.  Currently bisecting...

**Comment by janardhan1993** [comment]
@srh: Is this reproducible ?

**Comment by srh** [comment]
Yes, reliably.  Also I introduced the bug in 1dc69654e15dcc50113db7571635618c50202c4f.

**Comment by srh** [comment]
Fixed with a9c8a48ff39ae9a2d979afe53f6ed82e9a7fd05f.
